### DISCUSSION TITLE
Azure Monitor: Add support for example Log and ARG queries

### DISCUSSION THREAD

**Discussion by aangelisc** [grafana/grafana#60403] [Data Sources]
The experience in the Azure portal for Log Analytics and ARG includes example queries.

These queries could be invaluable to users that are first starting out with the datasource or are attempting to query a new resource and may be unaware of what columns and fields are supported. 

I propose adding these example queries to the Grafana UI to improve the experience for our users.

There is currently an [API](https://api.loganalytics.io/v1/metadata) that exposes example queries for Logs (and does not require auth).

There isn't a similar API for ARG, but there is documentation with some examples available which could be included.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
